whois-nrtm4-client/src/main/java/net/ripe/db/nrtm4/client/client/NrtmRestClient.java

```diff
@@ -89,7 +89,7 @@ public UpdateNotificationFileResponse getNotificationFile(final String source){
     @Nullable
     public byte[] getSnapshotFile(final String url){
         try {
-            final Response response =  client.target(url)
+            final Response response =  client.target(String.format("%s/%s", baseUrl, url))
```

Comment:
You don't need to concatenate the URL, but call the target with the `baseUrl` then add `.path(url)` and the client will construct the URL itself

Reply:
oh true, thanks